modules/ROOT/pages/am-installing.adoc

```diff
@@ -23,8 +23,8 @@ Ensure that Anypoint Monitoring is installed on a server running one of the foll
 
 * CentOS 7
 * RedHat Enterprise Linux 7.0 and 7.6
-* Ubuntu 14.04 and 15.04
-* Windows Server 2008 R2, Windows Server 2012, Windows Server 2012 R2, Windows Server 2016, Windows Server 2019
+* Ubuntu 16.04
```

Comment:
Is this 16.04 and later (current version is 18.04), or only 16.04?  Would be good to state either "and later" or "only".